x/community/keeper/rewards_test.go

```diff
@@ -0,0 +1,117 @@
+package keeper_test
+
+import (
+	"testing"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/stretchr/testify/require"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/kava-labs/kava/x/community/keeper"
+)
+
+func TestStakingRewardsCalculator(t *testing.T) {
```

Comment:
Tests are good and covers the edge cases well, but we aren't using any examples that match real world values or larger bounds.